app/assets/javascripts/modules/media_viewer.js

```diff
@@ -146,11 +147,22 @@
       return hlsSource.prop('src').indexOf('/mp3:') >= 0;
     }
 
+    function ms10Browser() {
+      var fingerprinter = new ClientJS();
+      var browser = fingerprinter.getBrowser();
+      return (((browser.toLowerCase() === 'edge') || (browser.toLowerCase() === 'ie')) &&
+               fingerprinter.getOS().toLowerCase() === 'windows' &&
+               fingerprinter.getOSVersion().toLowerCase() === '10');
+    }
+
     // We must use flash for MP3s on browsers that do not support HLS natively.
     // This is a simple boolean function that can easily tell us if the current
-    // browser does not support HLS and the media is an MP3
+    // browser does not support HLS and the media is an MP3. We also use flash
+    // on Windows 10 Microsoft browsers (IE and Edge) because of a known issue
+    // with the VideoJS HLS plugin.
```

Comment:
What would you think about including the link to the known-issues section (that was in the PR comment) here in the comment?

That way another developer can easily check that page from the code to see if the known-issue still exists (and they don't have to track the PR down for context).